runtime/help/options.md

```diff
@@ -462,6 +462,17 @@ Here are the available options:
 
     default value: `false`
 
+* `truecolor`: controls the usage of the true color support/mode.
+   * `auto`: will evaluate if the terminal is capable to handle true color.
+      This option can be overriden by setting `$MICRO_TRUECOLOR` to `1`.
+   * `on`: will force the usage of true color, but will need terminal support
```

Comment:
"force usage of true color even if micro does not detect its support by the terminal (of course this is not guaranteed to work well unless the terminal actually supports true color)"?